scripts/build-manifest.sh

```diff
@@ -30,20 +30,22 @@ main() {
 
   echo "${PASS}" | docker login -u "${USER}" --password-stdin
 
+  if [[ "${TRAVIS}" != "true" ]] || [[ "${TRAVIS_PULL_REQUEST}" != "false" ]] || [[ "${TRAVIS_BRANCH}" != "master" ]]; then
+    echo "****** Skipping manifest for: daily"
+    exit 0
+  fi
+
   echo "****** Building manifest for: daily"
   build_manifest "daily"
+}
 
+build_manifests() {
   local tags=$(git tag -l)
   while read -r tag; do
     if [[ -z "${tag}" ]]; then
       break
     fi
 
-    if [[ "${tag}" = "v0.0.1" ]]; then
```

Comment:
Even though releases are not built at the moment, we should retain this check. `v0.0.1` was the helm chart based operator, that we'll not rebuild.